Formula/sshguard.rb

```diff
@@ -16,12 +16,22 @@ class Sshguard < Formula
   depends_on "autoconf" => :build
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
+    system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
-                          "--with-firewall=#{firewall}"
+                          "--sysconfdir=#{etc}"
     system "make", "install"
+    mkdir "etc"
+    cp "examples/sshguard.conf.sample", "etc/sshguard.conf"
```

Comment:
Oh yes. I did this because by default it would look for the conf script in the Cellar directory. I changed the sysconfdir prefix to /usr/local, then forgot about this.